src/app/admin/admin.component.ts

```diff
@@ -178,7 +179,7 @@ export class AdminComponent implements OnInit {
     this._loadBuyDeSoFeeRate();
     this._loadUSDToDeSoReserveExchangeRate();
 
-    this.titleService.setTitle("Admin - DeSo");
+    this.titleService.setTitle(`Admin - ${environment.node.name}`);
```

Comment:
Would it be simpler if the `setTitle` function just appended the `${environment.node.name}` suffix to whichever title was provided?